int8 UNKNOWN = 0
int8 HOTSPOT = 1
int8 DISABLED = 2
int8 CONNECTED = 3
int8 DISCONNECTED = 4

int8 status